framework/src/functions/PiecewiseTabularBase.C

```diff
@@ -26,6 +27,10 @@ PiecewiseTabularBaseTempl<BaseClass>::validParams()
   params.addParam<FileName>("data_file", "File holding CSV data");
   params.addParam<unsigned int>("x_index_in_file", 0, "The abscissa index in the data file");
   params.addParam<unsigned int>("y_index_in_file", 1, "The ordinate index in the data file");
```

Comment:
We should also get rid of the 'in_file' part of the existing 'x_index_in_file' style parameters. That's kind of tangential, but it wouldn't be too hard to add an 'x_index' and 'y_index' parameter and deprecate the old ones. Maybe best for a follow-on PR.